src/docs/go-grpc/sections/03-operationalize/01-renewal/01-builtin.mdx

```diff
@@ -0,0 +1,93 @@
+The above example will work as long as the certificates do not expire. By
+default, `step-ca` certificates expire after 24 hours, so to go above that, we
```

Comment:
Do we want to call them `step-ca` certificates in PZT, or Certificate Manager certificates, or something else?
@alanchrt @mikemaxey

Reply:
Leave it without the modifier. Just "certificates"

Reply:
Or. Use. "smallstep certificates" expire...

Reply:
Fixed in 6022ce7